frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -50,25 +51,14 @@ const getPluginTabPages = (tabs: DashboardsTab[], cards: DashboardsCard[]): Page
 const DashboardsPage_: React.FC<DashboardsPageProps> = ({ match, kindsInFlight, k8sModels }) => {
   const { t } = useTranslation();
   const title = t('dashboard~Overview');
-  const tabExtensions = useExtensions<DashboardsTab>(isDashboardsTab);
-  const cardExtensions = useExtensions<DashboardsCard>(isDashboardsCard);
```

Comment:
You're removing a chance to add a new tab next to cluster overview completely. We still want that

Reply:
Thanks for your reviews. I have made the changes, PTAL.